apidef/oas/middleware_test.go

```diff
@@ -68,6 +68,27 @@ func TestPluginConfig(t *testing.T) {
 			assert.Equal(t, pluginConfig, newPluginConfig)
 		}
 	})
+
+	t.Run("bundle", func(t *testing.T) {
```

Comment:
Can you please add an empty case check?

Reply:
https://github.com/TykTechnologies/tyk/pull/4675/files#diff-0af31cb29ae298a6ac3e402b283ab364a6fd793fd04f253ef7c4983234c17befR36 - empty case is added here